addons/binding/org.openhab.binding.wifiled/src/test/java/org/openhab/binding/wifiled/handler/WiFiLEDHandlerTestApp.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.wifiled.handler;
```

Comment:
this should be move to a separate src folder (not in src/main/java, probably src/test/java), so that it does not end up in the production bundle code.